docs/run.md

```diff
@@ -4,11 +4,18 @@
 
 ### Prerequisites
 
-To use `clasp run`, you need to complete 3 steps:
+To use `clasp run`, you need to complete 4 steps:
 
 - Set up a **Project ID**.
 - Create an **OAuth Client ID** (Other). Download as `creds.json`.
 - `clasp login --creds creds.json` with this downloaded file.
+- Add the following to `appsscript.json`:
+  ```json
+  "executionApi": {
+    "access": "ANYONE"
+  }
+  ```
+- _(Optional) Setup Scopes_
```

Comment:
Can we remove this?
This section is for requirements.

Reply:
I honestly don't know. I discovered google appscripts exactly 3h before pushing this PR. 
All I know is:
* It was in the main `README.md`
* Without it `clasp run` wouldn't work on my machine

But I **do not** know:
* What it actually does
* If it is required for all project


Are you suggesting to remove it entirely from this document, or to move it to a different section?

I would just like to avoid a situation where newcomers would end up on this page (`/docs/run.md`) before going through the whole `README.md` and wouldn't know how to execute the command, like it happened to me.

Reply:
Ok.
Please remove this bullet since we don't add information here.

Users can find information under "Run a function that requires scopes"

Reply:
Removed :)